src/sidebar.js

```diff
@@ -311,22 +309,10 @@ module.exports = {
               type: "category",
               label: "Operations",
               items: [
-                "hydra/configure-deploy",
-                "hydra/dependencies-environment",
-                "hydra/production",
-                "hydra/guides/deploy-hydra-example",
-                "hydra/guides/tracing",
                 "hydra/guides/hsm-support",
-                "hydra/guides/secrets-key-rotation",
-                "hydra/guides/kubernetes-helm-chart",
-                "hydra/guides/ssl-https-tls",
                 "hydra/guides/cookies",
-                "hydra/guides/scaling-hydra",
                 "hydra/guides/cors",
-                "hydra/guides/gitlab",
-                "hydra/guides/migrating-from-mitreid",
-                "hydra/guides/merge-multiple-db-secrets",
-                "hydra/guides/upgrade",
+                ,
               ],
             },
             {
```

Comment:
Let's try to delete this "OAuth2 & OIDC" section as it's redundant with the name of the top level. Replace it with an "Advanced usage" section. 

-  "Creating OAuth2 Clients" -> rename to "Managing OAuth2 Clients", move to Guides section
- "Advanced ... flows" -> split into 3 pages: "OpenID Connect", "JSON Web Tokens (JWT)", and "Using Audiences". Merge the RFC7523 page into the JWT section (or put it next to it)
- "Using common OAuth2 and OIDC flows" -> rename to "Common flows", move under Guides and to the top
- "Using OAuth2" -> move to Guides, rename "OAuth2 client libraries"
- "Configure Token Experiation time" -> rename to "OAuth2 Token Lifetime", move to Configuration
- "OAuth 2.0 token introspection" -> leave here (under "advanced")
- "Mobile & browser (single-page-app) with OAuth2" -> Move to Guides, just under "Common flows", rename to "Mobile & Web Apps with OAuth2"
- "Updating claims at token refresh" -> leave here (under "advanced")